app/models/service/containers.rb

```diff
@@ -49,6 +50,19 @@ def create_container_object(wizard_state)
       container
     end
 
+    def container_attributes(wizard_state)
+      wizard_state.container_attributes.slice(:command, :repository_name,
```

Comment:
Need to figure out how to use permit here since `wizard_state.container_attributes` is a Hash.